packages/kbn-management/settings/application/application.tsx

```diff
@@ -43,6 +50,15 @@ function getQueryParam(url: string) {
   return '';
 }
 
+function getCategoryCounts(fields: FieldDefinition[]) {
```

Comment:
nit: I feel like this could be moved to a utility, to avoid this component becoming too bloated with static code.

Reply:
Thanks for the suggestion! I addressed this in https://github.com/elastic/kibana/pull/174301/commits/20a5e3bf9cc03286b09934cedb240acf17c17355.